proto/hub/hub.proto

```diff
@@ -14,18 +14,5 @@ message Hub {
 message RegisteredDenom {
   // base is the base of the denom
   string base = 1;
-
-  enum Status {
-    // PENDING is the status of the denom registration in the Hub
-    // when the denom has been sent to the Hub, but the dispute period has not yet ended.
-    PENDING = 0;
-    // ACTIVE is the status of the denom registration in the Hub
-    // when the denom has been sent to the Hub and the dispute period has ended.
-    ACTIVE = 1;
-    // INACTIVE is the status of the denom registration in the Hub
-    // when the denom registration had failed for whatever reason (malicious denom, timeout error, ack error, etc.)
-    INACTIVE = 2;
-  }
-  // status is the status of the denom registration in the Hub
-  Status status = 2;
+  reserved 2;
```

Comment:
we dont need it cause rollapp in productoin don't have it and we're gonna recreate the current ones.